submission/persistence-common-api/src/main/kotlin/ac/uk/ebi/biostd/persistence/common/service/SubmissionOperations.kt

```diff
@@ -100,10 +101,7 @@ interface SubmissionRequestPersistenceService {
      * Update the given request file. By default only file index is updated in submission request. For other options
      * @see UpdateOptions
      */
-    suspend fun updateRqtFile(
-        rqt: SubmissionRequestFile,
-        options: UpdateOptions? = null,
```

Comment:
Wouldn't this change render the `UpdateOptions` class deprecated? Perhaps we'll need to delete it